sprockets/mixins/amqp/__init__.py

```diff
@@ -0,0 +1,6 @@
+__import__('pkg_resources').declare_namespace(__name__)
+
+version_info = (0, 0, 1)
+__version__ = '.'.join(str(v) for v in version_info)
+
+__all__ = ('version_info', '__version__', 'RabbitMQRequestMixin')
```

Comment:
`RabbitMQRequestMixin` isn't an available symbol.  If you want to export a new symbol at the top-level so that a user can do `from sprockets.mixins.amqp import RabbitMQRequestMixin`, then you will need to import *amqp.py* here and export it.  Just be careful since this module is imported by *setup.py* when tornado is not installed.  IOW, do something like:
```python
try:
   from sprockets.mixins.amqp.mixins import RabbitMQRequestMixin
except ImportError as err:
   def RabbitMQRequestMixin(*args, **kwargs):
      raise err
```